### PR TITLE
UE bug fixes

### PR DESCRIPTION
**Pull request by TalmizAhmed**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes:

1. Increment and decrement options
2. Default value not updating in authoring

Test URLs:
- Before: https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/
- After: https://ue-bugs--aem-boilerplate-forms--adobe-rnd.hlx.live/

### CODE REVIEW

File: scripts/form-editor-support.js [hunk 1]
```diff
@@ -116,6 +116,16 @@ function cleanUp(content) {
   return formDef?.replace(/\x83\n|\n|\s\s+/g, '');
 }
 
+function decode(rawContent) {
```
Comment (jalagari) [inline scripts/form-editor-support.js:119]: We can export the function and reuse in editor.js instead of duplicating the code

**Review by nit23uec — APPROVED** [review]

File: scripts/form-editor-support.js (hunk 1 above)
Comment (nit23uec) [reply scripts/form-editor-support.js:119]: I would want to avoid any more contract between block and editor code.

File: blocks/form/form.js [hunk 2]
```diff
@@ -309,6 +309,9 @@ function inputDecorator(field, element) {
     if (field.maxFileSize) {
       input.dataset.maxFileSize = field.maxFileSize;
     }
+    if (field.default) {
+      input.value = field.default;
```
Comment (jalagari) [inline blocks/form/form.js:313]: @vdua, @nit23uec - For any changes in editor we are just calling  generateFormRendition which bypass createFormInstance due to which displayValue is not computed and value is coming empty whenever user change in dialog.

How are we handling such cases in Core Component?

Comment (nit23uec) [reply blocks/form/form.js:313]: we don't load the runtime view in core components authoring. so, the behavior is consistent wrt displayValue

File: scripts/form-editor-support.js (hunk 1 above)
Comment (jalagari) [reply scripts/form-editor-support.js:119]: Why? It currently depends on generateFormRendition which is the core the rendering so I don't see any reason of avoiding new dependence

Comment (nit23uec) [reply scripts/form-editor-support.js:119]: It's a tradeoff between having more contracts  and one line of code that needs to be duplicated (in future after far release is public, there would be just JSON.parse here). So I'll prefer not to expose any extra func from block code for this purpose.

**Pull request merged**